packages/amplify-e2e-tests/src/__tests__/auth-test-schemas/userpool-provider-fields.ts

```diff
@@ -0,0 +1,76 @@
+import { ImportedRDSType } from '@aws-amplify/graphql-transformer-core';
+import { convertToDBSpecificGraphQLString, generateDDL } from '../../rds-v2-test-utils';
+
+export const schema = (engine: ImportedRDSType): string => `
```

Comment:
Where is `engine` used?

Reply:
It's certainly is not used here, removed.